src/main/features/dashboard/views/contact-them.njk

```diff
@@ -0,0 +1,101 @@
+{% extends "layout.njk" %}
+{% from "./macro/caseHistory.njk" import caseHistoryForClaimantClaimDetails %}
+{% from "../../claim/views/macro/about-claim-widget.njk" import aboutClaimWidget %}
+
+{% from "table.njk" import tableStart,tableEnd, row, rowWithMultipleValue %}
+
+{% set headingVisible = false %}
+
+{% if user.id === claim.claimantId %}
+  {% set otherParty = 'defendant' %}
+  {% set otherPartyName = claim.claimData.defendant.name %}
+  {% set phone = claim.response.defendant.mobilePhone %}
+  {% set email = claim.claimData.defendant.email %}
```

Comment:
email should come from claim.defendantEmail if set, i.e. after we link the defendant the email will be set there, which covers the two cases of:
1. wrong email set by claimant
2. no email set initially

Reply:
done